gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -92,67 +85,40 @@ public void onViewCreated(
       @NonNull View view, @androidx.annotation.Nullable Bundle savedInstanceState) {
     super.onViewCreated(view, savedInstanceState);
     ((MainActivity) getActivity()).setActionBar(toolbar, R.drawable.ic_close_black_24dp);
+    // Initial state.
+    toolbar.setTitle("");
+    toolbar.setSubtitle("");
```

Comment:
Can we use [`startWith`](http://reactivex.io/documentation/operators/startwith.html) Instead of manually setting the value to ""?